### PR TITLE
movies project widgets migration

### PR DESCRIPTION
**Pull request by Kevin-Guillen**
In this project it can be found the widgets used in Movies app.

### CODE REVIEW

File: lib/src/widgets/movie_main_info.dart [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    required this.title,
+    required this.releaseDate,
+    required this.voteAverage,
+  }) : super(key: key);
+
+  final String movieImage;
+  final int id;
+  final String title;
+  final String releaseDate;
+  final num voteAverage;
```
Comment (FranciscoGaraventa) [inline lib/src/widgets/movie_main_info.dart]: This can be String, because inside this Widget you use toString() to parse it and that is not necessary here

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

File: lib/src/widgets/movie_main_info.dart [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    Key? key,
+    required this.movieImage,
+    required this.id,
+    required this.title,
+    required this.releaseDate,
+    required this.voteAverage,
+  }) : super(key: key);
+
+  final String movieImage;
+  final int id;
```
Comment (FranciscoGaraventa) [inline lib/src/widgets/movie_main_info.dart]: This variable is necessary?

Comment (Kevin-Guillen) [reply lib/src/widgets/movie_main_info.dart]: it was not necessary , deleted.

File: lib/src/widgets/movie_main_info.dart (hunk 1 above)
Comment (Kevin-Guillen) [reply lib/src/widgets/movie_main_info.dart]: Ok , fixed.

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**